dali/operators/audio/preemphasis_filter_op.cu

```diff
@@ -69,17 +69,18 @@ void PreemphasisFilterGPU::RunImplTyped(workspace_t<GPUBackend> &ws) {
   using SampleDesc = detail::SampleDescriptor<OutputType, InputType>;
   const auto &input = ws.InputRef<GPUBackend>(0);
   auto &output = ws.OutputRef<GPUBackend>(0);
+  auto curr_batch_size_ = ws.get_batch_size<GPUBackend>(0);
```

Comment:
This is not a member! Also, the "curr_" prefix serves very little purpose but to make the code _unnecessarily_ verbose.
```suggestion
  auto batch_size = ws.get_batch_size<GPUBackend>(0);
```